src/pet.js

```diff
@@ -6,6 +6,7 @@ function Pet(name) {
     this.hunger = 0;
     this.fitness = MAXIMUM_FITNESS;
 
+    // this should be a method on the prototype
```

Comment:
Here you're actually recreating this method in memory every time you create a new Pet, which is inefficient. It's better to have it on the `Pet`'s prototype so that each instance of Pet shares the same method.